requirements.txt

```diff
@@ -0,0 +1,2 @@
+--editable git+https://github.com/openfisca/openfisca-core.git@entity-to-json#egg=OpenFisca-Core
```

Comment:
This is required for tests to pass, while https://github.com/openfisca/openfisca-core/pull/425 is not merged. I remove it, and let's wait until 425 is merged.